features/ouders.feature

```diff
@@ -19,236 +19,234 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
+    | datumIngangFamilierechtelijkeBetrekking.type        | Datum        |
+    | datumIngangFamilierechtelijkeBetrekking.datum       | 2019-06-14   |
+    | datumIngangFamilierechtelijkeBetrekking.langFormaat | 14 juni 2019 |
 
   @gba
   Scenario: volledig onbekende ouder
-    Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende ouders in de registratie
-    | Categorie | Voornamen (02.10)  | Geslachtsnaam | Geboortedatum (03.10) | Geslachtsaanduiding (04.10) | Datum ingang familierechtelijke betrekking (62.10) |
-    | 2         |                    | .             |                       |                             | 00000000                                           |
-    | 3         |                    |               |                       |                             |                                                    |
-    En in categorie 3 komt alleen groep 81 en groep 85 voor
+    Gegeven de persoon met burgerservicenummer '000007897' heeft een ouder '1' met de volgende gegevens
+    | geslachtsnaam (02.40) | datum ingang familierechtelijke betrekking (62.10) |
+    | .                     | 00000000                                           |
+    En de persoon heeft een ouder '2' met de volgende gegevens
+	| aktenummer (81.20) | ingangsdatum geldigheid (85.10) |
+	| 2•E0001            | 20160518                        |
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                          |
     | type                | RaadpleegMetBurgerservicenummer |
-    | burgerservicenummer | 555550001                       |
+    | burgerservicenummer | 000007897                       |
     | fields              | ouders                          |
-    Dan bevat de persoon alleen ouders met de volgende gegevens
-    | naam                                    | waarde   |
-    | ouderAanduiding                         | 1        |
-    En heeft de ouder met ouderAanduiding '1' de volgende gegevens
-    | naam                                    | waarde   |
-    | ouderAanduiding                         | 1        |
-    | datumIngangFamilierechtelijkeBetrekking | 00000000 |
-    En de ouder met ouderAanduiding '1' heeft GEEN 'geslacht' gegevens
-    En de ouder met ouderAanduiding '1' heeft GEEN 'geboorte' gegevens
-    En heeft de ouder met ouderAanduiding '1' alleen de volgende 'naam' gegevens
-    | naam          | waarde |
-    | geslachtsnaam | .      |
+    Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
+    | naam                                                | waarde        |
+    | ouderAanduiding                                     | 1             |
+    | datumIngangFamilierechtelijkeBetrekking.type        | DatumOnbekend |
+    | datumIngangFamilierechtelijkeBetrekking.onbekend    | true          |
+    | datumIngangFamilierechtelijkeBetrekking.langFormaat | onbekend      |
```

Comment:
er moet ook een leeg naam object worden opgenomen. Dat staat nu niet beschreven. Als het goed is faalt dit scenario daarop.

Reply:
Lijkt mij in tegenspraak met de volgende rule in lege-waarden.feature:

```
  Rule: Een string (tekst) veld met lege waarde wordt niet geleverd
  Dit geldt ook voor een standaard/onbekendwaarde: zie onbekend_waardes.feature
```
en de volgende in onbekend_waardes.feature:

```
Rule: een veld wordt niet opgenomen wanneer het de standaardwaarde bevat
```